content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -37,22 +37,20 @@ kubectl:
 # start the pod running nginx
 kubectl create deployment --image=nginx nginx-app
 ```
-
-```shell
-# add env to nginx-app
-kubectl set env deployment/nginx-app  DOMAIN=cluster
 ```
 deployment.apps/nginx-app created
 ```
 
-```
+# add env to nginx-app
+kubectl set env deployment/nginx-app  DOMAIN=cluster
```

Comment:
Hello @msshroff .
Would you have look at lines 44-45? There appear to be duplicate commands listed.
Thanks!

Reply:
please ignore this PR. 
The issues are fixed by commit 82546d3ff88742d0209f78676f17a69f5ff03545 already.